zerver/tests/test_openapi.py

```diff
@@ -309,21 +382,24 @@ def test_openapi_arguments(self) -> None:
         for p in urlconf.v1_api_and_json_patterns:
             if p.lookup_str != 'zerver.lib.rest.rest_dispatch':
                 continue
+
+            # since the module was already imported and is now residing in
+            # memory, we won't actually face any performance penalties here.
             for method, value in p.default_args.items():
                 if isinstance(value, str):
-                    function = value
+                    function_name = value
                     tags = set()  # type: Set[str]
                 else:
-                    function, tags = value
+                    function_name, tags = value
+
+                lookup_parts = function_name.split('.')
+                module = __import__('.'.join(lookup_parts[:-1]), {}, {}, [''])
+                function = getattr(module, lookup_parts[-1])
 
                 # Our accounting logic in the `has_request_variables()`
                 # code means we have the list of all arguments
                 # accepted by every view function in arguments_map.
-                #
-                # TODO: Probably with a bit more work, we could get
-                # the types, too; `check_int` -> `int`, etc., and
-                # verify those too!
-                accepted_arguments = set(arguments_map[function])
+                accepted_arguments = set(arguments_map[function_name])
```

Comment:
Let's split out a preparatory refactor commit that just renames `function` to `function_name`.